### DISCUSSION TITLE
Meaning of representative elementary volume in a 2D medium?

### DISCUSSION THREAD

**Discussion by josephinoo** [PMEAL/porespy#546] [Q&A] [answered]
Hello, I have a question. If I apply representative_elementary_volume, to a medium generated in 2d dimensions, would it return representative elementary area or is it still representative_elementary_volume, since I am applying the metric in a 2D medium, this is a theoretical doubt. Thank you very much for your help.

**Comment by jgostick** [comment, marked as answer]
It was definitely written with 3D volumes in mind, but the code is dimension agnostic.  It calculates the porosity of each subsection, which in 2D means area fraction, rather than volume fraction.  Or you can think about it as if the 2D image has a 3rd dimension but the structure does not change in this dimension, like a cross-section through parallel cylinders will give you a 2D image of circles.  So I think the metric should be fine, but maybe to be rigorous refer to the y-axis as area fraction rather than porosity.

**Reply by josephinoo** [reply]
Thank you very much @jgostick
